dm/tests/dmctl_command/run.sh

```diff
@@ -152,6 +152,261 @@ function run() {
 	fi
 }
 
+function run_validation_start_stop_cmd {
+	cleanup_data dmctl_command
+	cleanup_data_upstream dmctl_command
+	cleanup_process $*
+
+	export GO_FAILPOINTS=""
+	run_dm_master $WORK_DIR/master $MASTER_PORT $cur/conf/dm-master.toml
+	check_rpc_alive $cur/../bin/check_master_online 127.0.0.1:$MASTER_PORT
+	run_dm_worker $WORK_DIR/worker1 $WORKER1_PORT $cur/conf/dm-worker1.toml
+	check_rpc_alive $cur/../bin/check_worker_online 127.0.0.1:$WORKER1_PORT
+	run_dm_worker $WORK_DIR/worker2 $WORKER2_PORT $cur/conf/dm-worker2.toml
+	check_rpc_alive $cur/../bin/check_worker_online 127.0.0.1:$WORKER2_PORT
+
+	run_sql_file $cur/data/db1.prepare.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1
+	run_sql_file $cur/data/db2.prepare.sql $MYSQL_HOST2 $MYSQL_PORT2 $MYSQL_PASSWORD2
+
+	dmctl_operate_source create $cur/conf/source1.yaml $SOURCE_ID1
+	dmctl_operate_source create $cur/conf/source2.yaml $SOURCE_ID2
+	dmctl_start_task $cur/conf/dm-task-no-validator.yaml --remove-meta
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"query-status test" \
+		"\"unit\": \"Sync\"" 2
+
+	echo "--> (fail) validation start: missing mode value"
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
```

Comment:
maybe also check the help message of validation xxx is readable, like `"pause-task \[-s source ...\] \[task-name | task-file\] \[flags\]"`

Reply:
no gains, cobra will just print cobra.Command.Use & cobra.Command.Short